README.md

```diff
@@ -167,6 +167,12 @@ There is then a single auth policy defined for e2e testing:
 
 ```sh
 curl -H "Host: test.a.auth.com" http://127.0.0.1:8000/get -i
+# HTTP/1.1 401 Unauthorized
+```
+
+```sh
+curl -H "Host: test.a.auth.com" -H "Authorization: APIKEY ndyBzreUzF4zqDQsqSPMHkRhriEOtcRx" http://127.0.0.1:8000/get -i
```

Comment:
That token is generated from our make targets?

Reply:
Yep - it's coming from here https://github.com/Kuadrant/wasm-shim/blob/external-auth/utils/deploy/authconfig.yaml#L27 (I took this one from the authorino repo).

I'm going to update this to use the `IAMALICE/BOB` in a future PR just for ease of use